package.json

```diff
@@ -26,17 +26,9 @@
     "@rollup/plugin-replace": "^6.0.1",
     "@rollup/plugin-typescript": "^12.1.0",
     "@types/jest": "^26.0.14",
-    "@types/ncp": "^2.0.4",
-    "@types/node": "^13.7.1",
-    "@types/node-fetch": "^2.5.7",
-    "@types/react": "^16.9.49",
-    "@types/react-dom": "^16.9.16",
-    "@types/react-native": "0.67.8",
```

Comment:
start to move dependencies out of the root package.json and have them defined by the packages that actually use them. This allows for various packages to use their own versions of things as necessary